src/components/settings/SettingsLMSTab/tests/ExistingLMSCardDeck.test.jsx

```diff
@@ -67,6 +69,11 @@ describe('<ExistingLMSCardDeck />', () => {
     );
     expect(screen.getByText('Incomplete')).toBeInTheDocument();
     expect(screen.getByText('barfoo')).toBeInTheDocument();
+    act(() => {
```

Comment:
I believe if you use the user-events library instead you will not need to use act() here (that may also be true of the built-in fireEvent but I am not sure). I have preferred using user-events when I can because it simluated the user events a bit more accurately wrt what happens in the browser

https://testing-library.com/docs/ecosystem-user-event#hoverelement-options

Reply:
we use it here, for example: https://github.com/edx/frontend-app-enterprise-public-catalog/blob/803d46471c857587a78eebd13cbb2f295169e22b/src/components/catalogSearchResults/buttons/downloadCsvButton/DownloadCsvButton.test.jsx